integration_test/ops_agent_test/main_test.go

```diff
@@ -2261,6 +2261,16 @@ func TestWindowsEventLogWithNonDefaultTimeZone(t *testing.T) {
 }
 
 func TestSystemdLog(t *testing.T) {
+	t.Parallel()
+	t.Run("fluent-bit", func(t *testing.T) {
+		testSystemdLog(t, false)
+	})
+	t.Run("otel", func(t *testing.T) {
+		testSystemdLog(t, true)
+	})
+}
+
+func testSystemdLog(t *testing.T, otel bool) {
```

Comment:
I'd like to see this actually test the schema of the log entries, instead of just testing that one exists.

Probably that means we should send a particular canary message to journald and then check how it is represented in Cloud Logging.